src/EnergyPlus/HVACManager.cc

```diff
@@ -381,11 +380,11 @@ namespace HVACManager {
 		NumOfSysTimeSteps = 1;
 		FracTimeStepZone = TimeStepSys / TimeStepZone;
 
+		SetOutAirNodes();
+
```

Comment:
humm, there are 4 EMS actuators for overriding out  node conditions, with this change the calling point will no longer work for those.   I think this will be a problem

Reply:
So move it back after the ManageEMS. @mjwitte why was this moved?

Reply:
@EnergyArchmage My (flawed) thinking here was if an outdoor air node is used as a sensor, it will see a zero is SetOutAirNodes hasn't happened yet.  But you're right, SetOutAirNodes won't respond to these until the next timestep (if at all?).  I'll put this one back.  

@rraustad hold on merging this.

But this makes me wonder if the timing of the new exterior conditions code in [InitHeatBalance](https://github.com/NREL/EnergyPlus/blob/001b3a8b6586a169ecaaab5666a8d77d53d15a18/src/EnergyPlus/HeatBalanceManager.cc#L4956-L4991) will work correctly - the only calling point I see before this is emsCallFromBeginNewEvironment.  The emsCallFromBeginTimestepBeforePredictor here seems too late to impact any surface heat balance - what am I missing here?  Seems like we need another calling point like emsCallFromBeginTimestepBeforeHeatBalance.

Reply:
in practice it often has been the case that emsCallFromBeginTimestepBeforePredictor doesn't work well, this InitHeatBalance timing may be the reason.  What does work, albeit with more lag, is to use the emsCallFromEndZoneTimestepAfterZoneReporting calling point to set things at the end of the previous timestep.  Might be another issue here, I don't think I ever looked at these new actuators before.

Reply:
Those particular actuators in InitHeatBalance are very new.  And there's an additional call to SetOutAirNodes there.